src/LazyImage/src/BlurHash/FetchImageContent.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\LazyImage\BlurHash;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final class FetchImageContent
```

Comment:
Is this necessary (genuine question) ? Is there a way to define the closure "file_get_content(...)" in the container ?

Reply:
At first I had this kind of approach in `BlurHash` constructor, but I felt like I had to create an invokable class instead:
```php
$this->fetchImageContent ??= fn (string $filename) => file_get_contents($filename);
```

> Is there a way to define the closure "file_get_content(...)" in the container?
I really don't know :/

Reply:
> `$this->fetchImageContent ??= fn (string $filename) => file_get_contents($filename);`

Let's go with this to keep this PR simpler.